backend/main.py

```diff
@@ -34,22 +34,22 @@
 
 # from pydantic import BaseModel
 
-
+MaxUsernameSize = 50 
+MaxPasswordSize = 200
```

Comment:
Constants are usually written in all capital letters with underscores separating words.